modules/serverless-rn-1-19-0.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-rn-1-19-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.19.0
+
+[id="new-features-1-19-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.25.
+* {ServerlessProductName} now uses Knative Eventing 0.25.
+* {ServerlessProductName} now uses Kourier 0.25.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.25.
+* {ServerlessProductName} now uses Knative Kafka 0.25.
+* The `kn func` CLI plug-in now uses `func` 0.19.
+
+* The `KafkaBinding` API is deprecated in {ServerlessProductName} 1.19.0 and will be removed in a future release.
+
+* HTTPS redirection is now supported and can be configured either globally for a cluster or per each Knative service.
+
+[id="fixed-issues-1-19-0_{context}"]
+== Fixed issues
+
+* In previous releases, the Kafka channel dispatcher waited only for the local commit to succeed before responding, which might have caused lost events in the case of an Apache Kafka node failure. The Kafka channel dispatcher now waits for all in-sync replicas to commit before responding.
+
+[id="known-issues-1-19-0_{context}"]
+== Known issues
+
+* In `func` version 0.19, some runtimes might be unable to build a function by using podman. You might see an error message similar to the following:
++
+[source,terminal]
+----
+ERROR: failed to image: error during connect: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.40/info": EOF
+----
++
+** The following workaround exists for this issue:
```

Comment:
Could just make this line a continuation of the parent item to reduce nesting, but NBD.